### DISCUSSION TITLE
How to print line in new line in airflow logs

### DISCUSSION THREAD

**Discussion by sant527** [apache/airflow#29799] [General]
I am using pythonoperator and using print("hello world") and this shows in the airflow logs like

```
[2023-02-28, 06:31:31 UTC] {logging_mixin.py:137} INFO -  hello wold
```

But i want it to be printed as below

```
[2023-02-28, 06:31:31 UTC] {logging_mixin.py:137} INFO -  
hello wold <-- in the next line not with the above line. because there are lof line i need to print and later copy them 
```

**Comment by potiuk** [comment]
same as any one other log in Python - and \n in your message

**Comment by rohan-prog-ux** [comment]
Hi, I would like to work on this issue as my first contribution. Can I take it?
